rfcs/0000-kuadrant_sub_components_configurations.md

```diff
@@ -0,0 +1,237 @@
+# Configuration of Kuadrant Sub Components
+
+- Feature Name: Configuration of Kuadrant Sub components.
```

Comment:
```suggestion
- Feature Name: `sub-components-config`
```

Reply:
What is the reasoning behind this change?

Reply:
Following the pattern from other RFCs. I think the idea for this is to be like a unique identifier that can be used to track the feature unequivocally across, e.g., file names, etc. It helps avoid overloading words and related confusion of such of when relying otherwise only on the human version of it (which is already in the title, BTW.)